### PR TITLE
Fix Recoveries edit form

### PR DESCRIPTION
**Pull request by Robertgervais**

### CODE REVIEW

**Comment by Robertgervais** [comment]
Image 6-14-19 at 1 18 PM

File: app/views/admin/recoveries/edit.html.haml [hunk 1]
```diff
@@ -64,19 +64,31 @@
         Recovery Description
       .card-body
         = @recovery.recovered_description
-%h2
-  - if @recovery.recovery_display.present?
-    = link_to "Recovery displayed", edit_admin_recovery_display_url(@recovery.recovery_display)
-  - else
-    - if @recovery.recovery_posted
-      Recovery posted
+/ %h2
```
Comment (sethherr) [inline app/views/admin/recoveries/edit.html.haml]: What's up with this commented out section?

**Comment by Robertgervais** [comment]
All good. There's still some more commented out view code on the bottom of `edit` that i'm not sure if you want to keep, otherwise everything is ~cool~

**Comment by Robertgervais** [comment]
#739

**Pull request merged**